.travis.yml

```diff
@@ -49,7 +41,3 @@ matrix:
 install: .travis/install.sh
 
 script: .travis/run.sh
-
-branches:
```

Comment:
This section should stay.

Reply:
Sure, reinstated. Out of curiosity, may I ask why? Thanks!

Reply:
One of the members of this org has a tendency to use the GH UI when submitting small patches. 😄  GH will make a branch on the primary rather than on a fork when you have write access to the primary. To avoid lots of extra CI runs we just keep this on all pyca projects. In the case of bcrypt it's not really a very big deal, but with cryptography it's painful to run CI multiple times for no reason.

Reply:
Ah, okay! That's exactly what we do when updating the changelog on another project, just edit through the UI, but here's a handy hint we use. Just put something like `[skip CI]` in the commit message, then it won't be built by Travis. :)

https://docs.travis-ci.com/user/customizing-the-build/#Skipping-a-build